dotnet/src/Agents/OpenAI/RunPollingOptions.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+using System;
+
+namespace Microsoft.SemanticKernel.Agents.OpenAI;
+
+/// <summary>
+/// Configuration and defaults associated with polling behavior for Assistant API run processing.
+/// </summary>
+public sealed class RunPollingOptions
```

Comment:
If this is not intented to be created or extended by the customer, suggest moving to internal as well as the constants, since they can't be overwritten or redefined.

```suggestion
internal sealed class RunPollingOptions
```